test/client/preparePayment/index.ts

```diff
@@ -364,7 +363,7 @@ export default <TestSuite>{
       `Flags = ${decoded.Flags}, should be 2147614720`
     )
     assert.deepEqual(result, expectedResult)
-    schemaValidator.schemaValidate('sign', result)
+    schemaValidate('sign', result)
```

Comment:
We shouldn't need to validate if `result` already equals `expectedResult`.

`expectedResult` should already be in the correct form.